Zend/tests/repeated_include/anonymous_class.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+Repeated include must not increase memory - anonymous class
+--SKIPIF--
+<?php
+// TODO test should pass even without opcache, but fixes are needed
+// related bug tracker https://bugs.php.net/bug.php?id=76982
+if (!extension_loaded('Zend OPcache') || !ini_get('opcache.enable') || !ini_get('opcache.enable_cli')) {
+    die('skip test currently requires opcache enabled');
+}
+?>
```

Comment:
I think you should use the `--EXTENSIONS--` section, and just check whether `opcache_get_status()["opcache_enabled"]` here.

Reply:
I have not added `--EXTENSIONS--` section by purpose to stress the fact opcache currently does not support it. I belive, the tests should pass without opcache in the future.

Yes, `opcache_get_status()["opcache_enabled"]` is better, thanks.